addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ParameterTamperScanRule.java

```diff
@@ -240,4 +237,39 @@ public int getCweId() {
     public int getWascId() {
         return 20;
     }
+
+    private AlertBuilder createAlert(int confidence, String param, String attack, String sb, HttpMessage msg) {
+        return newAlert()
+                .setConfidence(confidence)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(sb)
+                .setMessage(msg);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
+       alerts.add(
+            createAlert(
+                Alert.CONFIDENCE_MEDIUM,
+                "Example param",
+                "Example Attack",
+                "Example sb",
```

Comment:
Should use actual (realistic) example values.

Reply:
I am studying the solution, I've already made some changes you guys asked but I am kinda lost about the real examples. I looked here: https://www.zaproxy.org/docs/alerts/40008/ but no clue, could you help me, please?

Reply:
Look at the unit tests that trigger the alerts, you can temporarily add print statements to them to get realistic alert elements.